frontend/src/hooks/usePromptOutput.js

```diff
@@ -108,7 +109,7 @@ const usePromptOutput = () => {
         isSinglePass,
         true
       );
-
+      const coverageKey = `coverage_${llmProfile}_${item?.prompt_id}`;
```

Comment:
@jagadeeswaran-zipstack NIT: Since the `coverageKey` is generated in multiple places, consider creating a separate helper function for generating this string to improve code reuse and consistency with the pattern.